squape/object_tree.py

```diff
@@ -265,7 +263,7 @@ def _get_object_reference(object_name: any) -> any:
         object_name (any): symbolic name, real name, or object reference.
 
     Returns:
-        object_reference: Object reference from the given object_name.
+        (Squish object): Object reference from the given object_name.
```

Comment:
Will those changes be present in the sources too?

Reply:
I don't understand the question.

Reply:
I meant that would those changes to the docstrings also apply in the original files or will only remain in the documentation? I had a second look and it looks like those changes would be universal, so it's ok.